tools/bridge/tests/conftest.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from eth_tester import EthereumTester
+from web3 import Web3, EthereumTesterProvider
+
+
+@pytest.fixture
+def tester_foreign():
```

Comment:
```suggestion
def foreign_chain():
```

Reply:
Suggestion: I think this is more clear when used in the depths of upcoming tests. Also if fits with the naming of the `chain` fixture of the `contract-deploy-tools` plugin. Of cause the same for the home chain.

Reply:
As discussed, I still think `tester` is the better name as it refers to the class `EthereumTester` from the `eth-tester` library.